features/FxA-I-11/README.md

```diff
@@ -0,0 +1,128 @@
+# Match 1 - Control vs. No Show
+> Eye Password Experiment FXA-80
+
+Part of the [Eye Password Experiment](https://github.com/mozilla/fxa/blob/rfeeley/eye-password-emoji/features/FxA-80-eye-password-experiment/README.md)
+
+Idea in AHA: https://mozilla.aha.io/ideas/ideas/FXA-I-11
+
+## Outcomes
+
+The success of this feature is being able to answer the question:
+
+"Is the 'Show' password button useful in its current form?".
+"Does it reduce the number of errors the users make on the change password page?"
+
+## Assumptions
+
+- Users use this functionality on Android more.
+- We believe the "Show" password button is useful.
+
+## Problem Statement
+
+The change password flow can be improved in the current system.
```

Comment:
I'd suggest the basic problem statement is that we're not achieving my suggested outcome because we're observing a lot errors related to password (mis)-entry.